src/by/epam/ayem/module4/service/CountryService.java

```diff
@@ -0,0 +1,86 @@
+package by.epam.ayem.module4.service;
+
+/*3. Создать объект класса Государство, используя классы Область, Район, Город. Методы:
+вывести на консоль столицу, количество областей, площадь, областные центры.*/
+
+import by.epam.ayem.module4.model.City;
+import by.epam.ayem.module4.model.Country;
+import by.epam.ayem.module4.model.District;
+import by.epam.ayem.module4.model.Region;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class CountryService {
```

Comment:
You should have javadoc for all public classes as well as public methods

Reply:
There are no unit tests. All classes with business logic must be covered with unit tests.

Reply:
Got it! I'm going to add them later, once I'm done fixing the actual code in all the tasks.